pulp_rpm/app/advisory.py

```diff
@@ -123,6 +123,8 @@ def resolve_advisory_conflict(previous_advisory, added_advisory):
         previous_advisory.updated_date or previous_advisory.issued_date)
     added_updated_date = parse_datetime(
         added_advisory.updated_date or added_advisory.issued_date)
+    previous_updated_version = previous_advisory.version
```

Comment:
previous_updated_version = previous_advisory.version or ""?

Reply:
I think it is not needed as done on cr_to_dict (sync) or allow_empty on serializer (upload).